WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -233,17 +294,94 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         }
     }
 
-    /// Adds the sahre button at the right of the nav bar
+    private func configureNavigationBar() {
+        let rightItems = [
+            UIBarButtonItem.fixedSpace(24),
+            configuredMoreButton(),
+            UIBarButtonItem.fixedSpace(24),
+            configuredShareButton(),
+            UIBarButtonItem.fixedSpace(24),
+            configuredBrowserButton()
+        ]
+
+        navigationItem.leftBarButtonItem = configuredBackButton()
+        navigationItem.rightBarButtonItems = rightItems.compactMap({ $0 })
+    }
+
+    private func configuredBackButton() -> UIBarButtonItem {
+        let image = UIImage.gridicon(.chevronLeft).withRenderingMode(UIImage.RenderingMode.alwaysTemplate)
+        let button = CustomHighlightButton(frame: CGRect(x: 0, y: 0, width: image.size.width, height: image.size.height))
+        button.setImage(image, for: UIControl.State())
+        button.addTarget(self, action: #selector(didTapBackButton(_:)), for: .touchUpInside)
+
+        let barButtonItem = UIBarButtonItem(customView: button)
+        barButtonItem.accessibilityLabel = NSLocalizedString("Share", comment: "Spoken accessibility label")
+
+        return barButtonItem
+    }
+
+    @objc func didTapBackButton(_ sender: UIButton) {
+        navigationController?.popViewController(animated: true)
+    }
+
+    private func configuredBrowserButton() -> UIBarButtonItem? {
```

Comment:
```suggestion
    private func configureBrowserButton() -> UIBarButtonItem? {
```

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/commit/8479f0d1ec8913acb4007319d850a95b9f115828